### PR TITLE
Reduce logging on FileSystem instantiation

### PR DESCRIPTION
**Pull request by apc999**
Motivated by  issue https://github.com/Alluxio/alluxio/issues/11404, this PR updates logging level for certain logging messages 

After this PR, each query only introduced two info logging like the following;

```
2020-05-22T13:58:05.120-0700	INFO	hive-hive-6	alluxio.hadoop.AbstractFileSystem	Creating Alluxio configuration from Hadoop configuration Configuration: , uri configuration {}
2020-05-22T13:58:05.174-0700	INFO	20200522_205804_00001_nzuct.1.0.0-0-104	alluxio.hadoop.AbstractFileSystem	Creating Alluxio configuration from Hadoop configuration Configuration: , uri configuration {}
```

These two are required as currently each Presto query triggers a new Alluxio FileSystem to be created.

### CODE REVIEW

**Comment by apc999** [comment]
jenkins, test this please

File: core/client/hdfs/src/main/java/alluxio/hadoop/HadoopConfigurationUtils.java [hunk 1]
```diff
@@ -53,9 +53,7 @@ public static InstancedConfiguration mergeHadoopConfiguration(
         alluxioConfProperties.put(propertyName, entry.getValue());
       }
     }
-    LOG.info("Loading Alluxio properties from Hadoop configuration: {}", alluxioConfProperties);
```
Comment (apc999) [inline core/client/hdfs/src/main/java/alluxio/hadoop/HadoopConfigurationUtils.java:56]: we shall not `LOG.info` inside the util, moved to `AbstractFileSystem.initialize`

**Comment by apc999** [comment]
@calvinjia  PTAL

File: core/client/fs/src/main/java/alluxio/client/file/cache/LocalCacheManager.java [hunk 2]
```diff
@@ -97,10 +97,9 @@
    */
   private static boolean restore(PageStore pageStore, PageStoreOptions options, MetaStore metaStore,
       CacheEvictor evictor) {
-    LOG.info("Restore PageStore with {}", options);
```
Comment (calvinjia) [inline core/client/fs/src/main/java/alluxio/client/file/cache/LocalCacheManager.java:100]: Is this line too noisy? I think its helpful to see it

Comment (apc999) [reply core/client/fs/src/main/java/alluxio/client/file/cache/LocalCacheManager.java:100]: reverted with updated message

**Comment by apc999** [comment]
@calvinjia  PTAL

**Comment by apc999** [comment]
alluxio-bot, merge this please

**Pull request merged**